src/screens/gratitudeEntry/TagsModal.tsx

```
@@ -0,0 +1,352 @@
+
+          {viewState === 'select' ? renderSelectView() : renderFormView()}
+        </View>
+      </BottomSheet>
+
+      {/* Delete Confirmation Dialog */}
+      <AlertDialog open={deleteDialogOpen} onOpenChange={setDeleteDialogOpen}>
+        <AlertDialogContent>
+          <AlertDialogHeader>
+            <AlertDialogTitle>{t('Delete Tag')}</AlertDialogTitle>
+            <AlertDialogDescription>
+              {t('Are you sure you want to delete the tag?')} "{tagToDelete?.title}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)